ogarniacz_mp3.py

```diff
@@ -72,14 +78,18 @@ def processing_to_verification(processing_dir_path):
                                 os.remove(file_path)
                             except FileNotFoundError:
                                 continue # skoro tego pliku nie ma, to nie ma co tutaj robić
-
-    # b) przekonwertować nie mp3 na mp3
-    # c) dostosować bitrate'y tam, gdzie to konieczne
-    # d) dostosować poziom głośności
-    # e) znormalizować tagi
-    # f) pozbyć się zbędnych tagów
-    # 2. przeniesienie ich do verification
-    pass
+                            # b) przekonwertować nie mp3 na mp3
+                            if ext != "mp3":
+                                pass
+                            # c) dostosować bitrate'y tam, gdzie to konieczne
+                            # d) dostosować poziom głośności
+                            # e) znormalizować tagi
+                            # f) pozbyć się zbędnych tagów
+                            # 2. przeniesienie ich do verification
+                            try:
+                                shutil.move(file_path, destination_pth)
+                            except FileNotFoundError:
+                                continue #jeśli pliku nie ma, to nie można go przenieść
```

Comment:
Wróć z tym wszystkim do poprzedniego poziomu wcięcia - trzeba będzie na to zrobić oddzielną pętlę.
Zwróć uwagę na wcięcia i warunki które muszą być spełnione, żeby ten kod się uruchomił. Najpierw próbujesz usunąć plik (`os.remove(file_path)`) a potem go przenieść (`shutil.move(file_path, destination_pth`).